### DISCUSSION TITLE
Unitalicizing footnotes in pull quotes?

### DISCUSSION THREAD

**Discussion by mmpadilla15** [thegetty/quire#576] [Get Help] [answered]
Is there a way to unitalicize the footnote numbers in pull quotes?

<img width="664" alt="Screenshot 2022-11-04 at 12 59 21 PM" src="https://user-images.githubusercontent.com/74433007/200044537-57f26ab8-d4d6-452f-8315-3a9cac7c784a.png">

**Comment by Erin-Cecele** [comment, marked as answer]
Yes, you can use Custom CSS. Try adding this to your `custom.css` file and let me know if it works:
```
.quire-page__content .content .footnote-ref a, .quire-page__content .content a.footnote-return, .quire-page__content .content a.footnote-backref, .quire-page__content .content a.footnote-ref {
  font-style: normal;
}
```

That might not be quite right but let's give it a shot. 

Also, just a note that in the new 11ty version of Quire the footnotes in blockquotes will appear roman rather than italicized, so we can avoid this in the future.

**Reply by mmpadilla15** [reply]
@Erin-Cecele It worked! And I didn't have to hard refresh!

**Reply by Erin-Cecele** [reply]
Excellent! 

If we offered a tutorial on custom CSS like this when the new version is released, would you be interested?

**Reply by mmpadilla15** [reply]
@Erin-Cecele Absolutely!
